src/stories/beta/Form.js

```diff
@@ -145,12 +153,23 @@ storiesOf('Beta/Form', module)
     )
   })
   .add('UI schema', () => {
+    const formData = {
+      poke_password: 'Secret password'
+    }
+
     return (
       <Provider>
         <FormDemo
+          formData={formData}
           schema={basicPokedexSchema}
           uiSchema={{
-            'ui:order': ['Name', 'caught', 'Description', 'Abilities', '*']
+            'ui:order': ['Name', 'caught', 'Description', 'Abilities', '*'],
+            poke_password: {
+              'ui:widget': 'password',
+              'ui:options': {
+                showPasswordStrengthMeter: true
```

Comment:
This options seems to be unused

Reply:
It should be used as the strength meter should be opt-in, but I removed it at one point and forgot to put it back again. Will fix it right away.